ansible/roles/operator-pipeline/templates/openshift/tasks/preflight-trigger.yml

```diff
@@ -4,15 +4,12 @@ metadata:
   name: preflight-trigger
 spec:
   params:
-    - name: fedora_image
-      description: Fedora image
-      default: "docker.io/library/fedora"
-    - name: go_toolset_image
-      description: GO toolset image
-      default: "registry.access.redhat.com/ubi8/go-toolset"
     - name: preflight_trigger_image
       description: Preflight trigger image
-      default: "quay.io/opdev/preflight-trigger"
+      default: "quay.io/opdev/preflight-trigger:stable"
+    - name: preflight_trigger_environment
```

Comment:
Since you are changing task params you should also update a hosted pipeline to pass the correct parameters and also update the pipeline trigger and pass an ansible variable as pipeline parameter.

Reply:
I am not clear on the ask here since this one was not being passed in before.